vm/src/function/protocol.rs

```diff
@@ -100,6 +101,13 @@ where
     }
 }
 
+impl From<ArgIterable> for PyObjectRef {
```

Comment:
I don't think this is necessary for your approach.
Instead of this one, maybe `impl From<AnySet> for PyObjectRef` could be useful.

Reply:
Thank you for your advice.
I'll give it a try!